.github/workflows/ci.yml

```diff
@@ -90,7 +90,10 @@ jobs:
       - name: 'Bazel: fetch'
         run: bazel fetch //tensorboard/...
       - name: 'Bazel: build'
-        run: bazel build //tensorboard/...
+        run: |
+          ( set -o pipefail;
+            bazel build //tensorboard/... |&\
+            grep -v 'external/com_google_protobuf/python: warning: directory' )
```

Comment:
This does leave a bunch of dangling messages like:

INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/any_pb2.py [for host]:
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/source_context_pb2.py [for host]:

Which might be confusing given that we're not acctually printing out the associated warnings from these files. Is there a way to also suppress these lines when the only associated error message is the one you are currently trying to suppress?